inc/class-taxonomy.php

```diff
@@ -649,4 +552,31 @@ protected function upgradeChapterTypes() {
 			wp_delete_term( $type_5->term_id, 'chapter-type' );
 		}
 	}
+
+	/**
+	 * Upgrade to new Contributor Data Model
+	 *
+	 * @param int $meta_id ID of updated metadata entry.
+	 * @param int $post_id Post ID.
+	 * @param string $old_meta_key Meta key.
+	 * @param mixed $meta_value Meta value.
+	 *
+	 * @return array|false An array containing the `term_id` and `term_taxonomy_id`, false otherwise.
+	 */
+	public function upgradeToContributorTaxonomy( $meta_id, $post_id, $old_meta_key, $meta_value ) {
+		$contributor_migration = [
+			'pb_author' => 'pb_authors',
+			'pb_section_author' => 'pb_authors',
+			'pb_editor' => 'pb_editors',
+			'pb_translator' => 'pb_translators',
+			'pb_contributing_authors' => 'pb_contributors',
+		];
+		if ( isset( $contributor_migration[ $old_meta_key ] ) && ! empty( $meta_value ) && is_string( $meta_value ) ) {
```

Comment:
As per above, `$meta_value` should be an array for `pb_editor`, `pb_translator`, and `pb_contributing_authors` so we would need to iterate over it.